lib/mixins/pdfa.js

```diff
@@ -0,0 +1,110 @@
+export default {
+
+    initPDFA(pOptions) {
+        if (pOptions.pdfa) {
```

Comment:
Maybe this option should be called `profile`, in case we want to add support for other PDF profiles like `PDF/E`, `PDF/UA`, etc. in the future? Could accept a string like `profile: 'PDF/A-1'` etc.

Reply:
Great idea - I can see us needing support for PDF/UA very soon and would be nice to support PDF/E and other subsets in the future.

Would you be happy with `profile` or `subset`/`pdfsubset`?

Also, would you prefer receiving strings or shall we have identifiers and therefore pass something along the lines of `PDFDocument.Subsets.PDFA1a`/`PDFDocument.Subsets.PDFA1b`/`PDFDocument.Subsets.PDFE`? I'm thinking they might be cleaner than strings, plus if we ever come across a case where a PDF can be PDF/A and PDF/E compliant, we can pass `PDFDocument.Subsets.PDFA1a+PDFDocument.Subsets.PDFE`?

Reply:
Personally, I prefer strings for brevity, but we could also support an array if multiple can be supported at once. I don't have a strong preference on the name. Subsets are fine.

Reply:
Added a new commit to support `subset` as well as a couple mixins to control metadata and load subsets at runtime, if you're happy I can continue and add the documentation for this?

Reply:
Docs should now refer to `subset` instead of the old `pdfa` :)

Is there anything else I should check/change or is this good to merge?